devlab/tests/generate_load.py

```diff
@@ -403,6 +403,8 @@ def wait_until_images_created(image_ids):
                         member_id = tenant['id']
                         self.glanceclient.image_members.create(image_id,
                                                                member_id)
+        if getattr(self.config, 'create_zero_image', None):
```

Comment:
getattr is used instead of self.config.create_zero_image to make create_zero_image  parameter optional in config.py